include/staff/helptopic.inc.php

```diff
@@ -93,6 +99,34 @@
                 &nbsp;<span class="error">&nbsp;<?php echo $errors['pid']; ?></span>
             </td>
         </tr>
+        <tr>
+            <td width="180">
+                <?php echo __('Allowed Organizations');?>:
+            </td>
+            <td>
+                <select name="organizations[]" class="multi-select" multiple>
+                    <?php $allOrganizations = Organization::getAllOrganizations();
+                    while (list($id,$name) = each($allOrganizations)) {
+                        if ($id == $info['id'])
```

Comment:
I believe this `if` on row 110 and 111 could be removed, since `$allOrganizations` only contain unique entries (from comment above).